api.js

```diff
@@ -1,12 +1,24 @@
 import axios from 'axios';
 
-export default function getArticles() {
+function getArticles() {
     return axios
     .get("https://my-nc-news-e0va.onrender.com/api/articles")
     .then((result) => {
-        return result.data
+        return result.data;
     })
     .catch(() => {
-        return Promise.reject()}
-    )
-}
\ No newline at end of file
+        return Promise.reject();
```

Comment:
When you catch and promise reject, you should probably give some indication as to what went wrong